### PR TITLE
Style updates for  Scalability and performance guide

### PR DESCRIPTION
**Pull request by ahardin-rh**
Note that the **OpenShift Container Platform tested cluster maximums table** will be updated separately for each product version in other PRs, but this should catch the rest.

### CODE REVIEW

File: modules/topology-manager-policies.adoc [hunk 1]
```diff
@@ -21,27 +21,16 @@ This is the default policy and does not perform any topology alignment.
...
 `restricted` policy::
 
-For each container in a Pod with the `restricted` topology
-management policy, kubelet calls each Hint Provider to discover their resource
-availability. Using this information, the Topology Manager stores the
-preferred NUMA Node affinity for that container. If the affinity is not
-preferred, Topology Manager rejects this pod from the node,
-resulting in a pod in a Terminated state with a pod admission failure.
+For each container in a pod with the `restricted` topology management policy, kubelet calls each Hint Provider to discover their resource
+availability. Using this information, the Topology Manager stores the preferred NUMA Node affinity for that container. If the affinity is not
```
Comment (jeana-redhat) [inline modules/topology-manager-policies.adoc:31]: this one's hard-wrapped

File: scalability_and_performance/using-cluster-loader.adoc [hunk 2]
```diff
@@ -5,10 +5,8 @@ include::modules/common-attributes.adoc[]
 
 toc::[]
 
-Cluster Loader is a tool that deploys large numbers of various objects to a
-cluster, which creates user-defined cluster objects. Build, configure, and run
-Cluster Loader to measure performance metrics of your {product-title} deployment
-at various cluster states.
+Cluster Loader is a tool that deploys large numbers of various objects to a cluster, which creates user-defined cluster objects. Build, configure, and run
+Cluster Loader to measure performance metrics of your {product-title} deployment at various cluster states.
```
Comment (jeana-redhat) [inline scalability_and_performance/using-cluster-loader.adoc:9]: this one's hard-wrapped

File: scalability_and_performance/using-cluster-loader.adoc [hunk 3]
```diff
@@ -19,11 +17,9 @@ include::modules/configuring-cluster-loader.adoc[leveloffset=+1]
...
 == Known issues
 
-* Cluster Loader fails when called without configuration.
-(link:https://bugzilla.redhat.com/show_bug.cgi?id=1761925[*BZ#1761925*])
+* Cluster Loader fails when called without configuration. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1761925[*BZ#1761925*])
 
-* If the `IDENTIFIER` parameter is not defined in user templates, template
-creation fails with `error: unknown parameter name "IDENTIFIER"`. If you deploy
+* If the `IDENTIFIER` parameter is not defined in user templates, template creation fails with `error: unknown parameter name "IDENTIFIER"`. If you deploy
 templates, add this parameter to your template to avoid this error:
```
Comment (jeana-redhat) [inline scalability_and_performance/using-cluster-loader.adoc:23]: this one's hard-wrapped

**Review by jeana-redhat — APPROVED** [review]
Saw some places you undid hard wraps and some places not - so I wasn't sure if you had it in scope for this round. Feel free to ignore where I pointed those out. Overall looks great, a few missed items and some I'm unsure about.

File: modules/accessing-an-example-cluster-node-tuning-operator-specification.adoc [hunk 4]
```diff
@@ -17,20 +17,9 @@ Use this process to access an example Node Tuning Operator specification.
 $ oc get Tuned/default -o yaml -n openshift-cluster-node-tuning-operator
 ----
 
-The default CR is meant for delivering standard node-level tuning for the
-{product-title} platform and it can only be modified to set the Operator
-Management state. Any other custom changes to the default CR will be
-overwritten by the Operator. For custom tuning, create your own Tuned CRs. Newly
-created CRs will be combined with the default CR and custom tuning applied to
-{product-title} nodes based on node or Pod labels and profile priorities.
+The default CR is meant for delivering standard node-level tuning for the {product-title} platform and it can only be modified to set the Operator Management state. Any other custom changes to the default CR will be overwritten by the Operator. For custom tuning, create your own Tuned CRs. Newly created CRs will be combined with the default CR and custom tuning applied to {product-title} nodes based on node or Pod labels and profile priorities.
```
[the remark this answers is not in the document]
Comment (ahardin-rh) [reply modules/accessing-an-example-cluster-node-tuning-operator-specification.adoc:20]: According to Jiri, Tuned should be capitalized

**Pull request merged**